tv-shows-app/src/components/features/shows/ShowDetails/ShowDetails.tsx

```diff
@@ -35,7 +37,7 @@ export default function ShowDetails({
       </Heading>
       <Text paddingX={5}>{description}</Text>
       <Text paddingX={5} paddingBottom={5}>
-        {averageRating ? averageRating.toFixed(2) + " / 5" : "No ratings"}
+        {average_rating ? average_rating.toFixed(2) + " / 5" : "No ratings"}
```

Comment:
The way you named it initially with camelCase is actually what we have in our styleguide.

Reply:
I see now that that is what the mock_data provides, so please ignore the comment.